raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -912,7 +914,6 @@ public void poll() throws IOException {
         if (gracefulShutdown != null) {
             pollShutdown(gracefulShutdown);
         } else {
-            // This call is a bit annoying, but perhaps justifiable if we need to acquire a lock
```

Comment:
It's not really, so I just removed the comment. Probably I was annoyed initially because it makes another call to `time.milliseconds()` under the covers.